diff --git a/src/realm.h b/src/realm.h
index 8c7f381ad..efa5a02fb 100644
--- a/src/realm.h
+++ b/src/realm.h
@@ -3152,7 +3152,7 @@ RLM_API realm_flx_sync_subscription_t* realm_sync_subscription_at(const realm_fl
  *  @return a pointer to the subscription with the name passed as parameter
  */
 RLM_API realm_flx_sync_subscription_t* realm_sync_find_subscription_by_name(const realm_flx_sync_subscription_set_t*,
-                                                                        const char*) RLM_API_NOEXCEPT;
+                                                                            const char*) RLM_API_NOEXCEPT;
 /**
  *  Find subscription associated to the query passed as parameter
  *  @return a pointer to the subscription associated to the query passed
@@ -3176,8 +3176,9 @@ realm_sync_make_subscription_set_mutable(realm_flx_sync_subscription_set_t*) RLM
  */
 RLM_API bool realm_sync_subscription_set_clear(realm_flx_sync_mutable_subscription_set_t*) RLM_API_NOEXCEPT;
 /**
- *  Insert ot update a query for the subscription set passed as parameter, if successful the index where the query was inserted or updated is returned along
- *  with the info whether a new query was inserted or not. It is possible to specify a name for the query inserted
+ *  Insert ot update a query for the subscription set passed as parameter, if successful the index where the query was
+ * inserted or updated is returned along with the info whether a new query was inserted or not. It is possible to
+ * specify a name for the query inserted
  *  @return true/false if operation was successful
  */
 RLM_API bool realm_sync_subscription_set_insert_or_assign(realm_flx_sync_mutable_subscription_set_t*, realm_query_t*,
